examples/example-next-static-export/README.md

```diff
@@ -0,0 +1,11 @@
+# example-next-static-export
```

Comment:
Do we want to add to the READMEs how to use/run these examples?

Reply:
Hmm, the thing is that there's only so much that you can do with the example because it needs a real ontology and stack to login and start loading data. I agree it would be nice if there was something more to see do you have any ideas? Ideally it stays as close as possible to the stock rendered template so we can keep it in sync automatically and it serves as our end to end test that the template is buildable